scripts/fuzz.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Fuzzing tests used by CI. Free feel to modify or remove `RUNS` for local development or local tests.
+
+set -euxo pipefail
+
+RUNS=1000000
```

Comment:
Since the project and github actions are both using `cargo-make` you could actually implement this there for cross platform compatibility, self installing cargo extensions etc. To do this:

To the top of `Makefile.toml`:
```toml
[env]
FUZZ_RUNS = 1000000
```

In the tasks section of `Makefile.toml`:
```toml
[tasks.fuzz]
dependencies = [
    "fuzz-arithmetic",
    "fuzz-constructors"
]

[tasks.fuzz-arithmetic]
toolchain = "nightly"
install_crate = "cargo-fuzz"
command = "cargo"
args = ["fuzz", "run", "arithmetic", "--", "-runs=$FUZZ_RUNS"]

[tasks.fuzz-constructors]
toolchain = "nightly"
install_crate = "cargo-fuzz"
command = "cargo"
args = ["fuzz", "run", "constructors", "--", "-runs=$FUZZ_RUNS"]
```

You _should_ be able to just run `makers fuzz` to run everything, or if you wanted to isolate to just a portion of tests you'd run `makers fuzz-constructors`. 

It's where I've added all the test configurations, benchmarking logic as well as code coverage rules - so it'd be nice to include fuzz in there too.

To finish off - this is why I need to write a `BUILDING.md` document or whatever. Another vote for #380 !